src/components/Checkout/DonationItem.vue

```diff
@@ -9,15 +9,25 @@
 			<span class="donation-info featured-text">
 				{{ donationTitle }}
 			</span>
-			<div>
-				<div v-if="hasLoans" class="donation-tagline small-text">{{ donationTagLine }}</div>
-				<a
-					class="small-text donation-help-text"
-					@click.prevent="triggerDefaultLightbox"
-					v-kv-track-event="['basket', 'Donation Info Lightbox', 'Open Lightbox']"
+			<div v-if="hasLoans">
+				<div
+					v-if="specificDonationUseTextExperiment"
+					class="donation-tagline small-text"
 				>
-					{{ donationDetailsLink }}
-				</a>
+					{{ specificDonationTagLine }}
```

Comment:
Could also just hardcode the experiment text here if that would be prefered?

Reply:
Yeah since it's just static text go ahead and hardcode it here

Reply:
✔️